pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -336,6 +336,23 @@ class Timestamp(_Timestamp):
         """
         return cls(datetime.combine(date, time))
 
+    @classmethod
+    def fromisocalendar(cls, year, week, day):
+        """
+        Timestamp.fromisocalendar(year, week, day)
+
+        Return a new Timestamp corresponding to the
+        ISO calendar date specified by year, week and day.
+        """
+        import pandas.compat as compat
+
+        if not compat.PY38:
+            raise NotImplementedError(
+                "'fromisocalendar' is not supported for versions earlier to 3.8"
```

Comment:
`"'fromisocalendar' is not supported for versions earlier than 3.8"`

Reply:
@mroeschke This is intended.

Do you have any suggestions?

(Grammer/Implementation/etc..?)

Reply:
`earlier to 3.8"` -> earlier than 3.8"

Reply:
Ah I see it now, TY:)